### DISCUSSION TITLE
Support for building Multilingual applications

### DISCUSSION THREAD

**Discussion by YogiYang** [twinbasic/twinbasic#792] [General]
In case of VB6 it is a nightmare to build multilingual applications.

Are there any plans of adding easy to understand and use support for Multilingual features?

**Comment by KDGundermann** [comment]
Sorry to say, but it is not a nightmare.
As in every other language what you need is a library for storing tagged strings and a function which "translates" a source string (or constant) to a localized string.
Really easy to implement.
